draft-ietf-webpush-protocol.xml

```diff
@@ -799,6 +925,16 @@ HEADERS      [stream 4] +END_STREAM
           superceded (e.g. the current number of unread emails).
         </t>
       </section>
+      <section anchor="sets-streams" title="Subscription Sets and Concurrent HTTP/2 streams">
+        <t>
+          If the push service requires that the user agent use push message subscription sets,
+          then it SHOULD limit the number of concurrently active streams with the
+          SETTINGS_MAX_CONCURRENT_STREAMS parameter within a HTTP/2 SETTINGS frame <xref target="RFC7540"/>. 
+          
+          The user agent SHOULD be limited to one concurrent stream to manage push message
+          subscriptions and one concurrent stream per subscription set returned by the push service.

```

Comment:
Nit: Is this two paragraphs or one?

I think that MAY is more appropriate than SHOULD.  You should also note that this choice limits the concurrency available for managing subscriptions.